packages/data-context/src/codegen/templates/e2e/e2e.ejs

```diff
@@ -3,7 +3,7 @@ fileName: <%= fileName %>
 ---
 
 describe('<%= fileName %>', () => {
-  it('should visit', () => {
-    cy.visit('/')
+  it('should', () => {
+    expect(true).eq(true)
```

Comment:
why not keep visit? 
it feels more cypress e2e.

Reply:
I agree, will revert